pyproject.toml

```diff
@@ -131,9 +131,12 @@ feff_plot_cross_section = "pymatgen.cli.feff_plot_cross_section:main"
 feff_plot_dos = "pymatgen.cli.feff_plot_dos:main"
 get_environment = "pymatgen.cli.get_environment:main"
 
+[tool.setuptools]
+include-package-data = false
+
 [tool.setuptools.packages.find]
```

Comment:
We can completely get rid of this `[tool.setuptools.packages.find]` section completely owing to the [`automatic discovery` feature of `setuptools`](https://setuptools.pypa.io/en/latest/userguide/package_discovery.html), but I guess explicitly declaring it might be clearer?